src/Serilog/Core/Pipeline/SilentLogger.cs

```diff
@@ -20,6 +20,8 @@ namespace Serilog.Core.Pipeline
 {
     class SilentLogger : ILogger
     {
+        public readonly static ILogger None = new SilentLogger();
```

Comment:
Declaration order would normally be `public static readonly` according to the conventions of this codebase.

The property name `Instance` would possibly be more appropriate here?